lib/v2/datasource/remote/api/eos_repository.dart

```diff
@@ -33,7 +33,7 @@ abstract class EosRepository {
   EOSClient buildEosClient(String nodeEndpoint, String privateKey) =>
       EOSClient(nodeEndpoint, 'v1', privateKeys: [privateKey, cpuPrivateKey]);
 
-  Result mapSuccess(dynamic response, Function modelMapper) {
+  Result mapEosSuccess(dynamic response, Function modelMapper) {
```

Comment:
Can this be just mapSuccess?

Reply:
I put the reasons in "Notes for Code Reviewer". 

Rename the Eos Repository mappers because there are conflicts in the ProfileRepository on which mapper to use (Network, Eos).